### PR TITLE
change: Use loading icon to indicate tax id is being verified

### PR DESCRIPTION
**Pull request by jaalah-akamai**
## Description 📝
We need to verify the tax id before any of the events `tax_id_valid` or `tax_id_invalid` are fired off. This new icon provides a better UX.

See for more details: https://github.com/linode/manager/pull/10558

## Changes  🔄
- Removed conditional icon for tax id

## Target release date 🗓️
9/16

## Preview 📷
**Include a screenshot or screen recording of the change**

https://github.com/user-attachments/assets/e2a9c20d-54ae-446b-a853-e4148c41973b

| Before  | After   |
| ------- | ------- |
| Screenshot 2024-07-08 at 9 59 11 AM | Screenshot 2024-09-12 at 3 08 30 PM|

### Verification steps
Just observe that removing code does not break anything within the file itself.

## As an Author I have considered 🤔

*Check all that apply*

- [x] 👀 Doing a self review
- [ ] ❔ Our [contribution guidelines](https://github.com/linode/manager/blob/develop/docs/CONTRIBUTING.md)
- [ ] 🤏 Splitting feature into small PRs
- [x] ➕ Adding a [changeset](https://github.com/linode/manager/blob/develop/docs/CONTRIBUTING.md#writing-a-changeset)
- [ ] 🧪 Providing/Improving test coverage
- [ ] 🔐 Removing all sensitive information from the code and PR description
- [ ] 🚩 Using a feature flag to protect the release
- [ ] 👣 Providing comprehensive reproduction steps
- [ ] 📑 Providing or updating our documentation
- [ ] 🕛 Scheduling a pair reviewing session
- [ ] 📱 Providing mobile support
- [ ] ♿  Providing accessibility support

### CODE REVIEW

**Review by coliu-akamai — APPROVED** [review]
didn't see anything breaking in the file 👍

**Review by bnussman-akamai — APPROVED** [review]
Does the API just clear the account's `tax_id` field if it is invalid? Wondering if it's bad that were just cleaning user data

**Comment by bnussman-akamai** [comment]
Understood! Thanks @jaalah-akamai

**Review by hkhalil-akamai — APPROVED** [review]

File: packages/manager/src/queries/account/account.ts [hunk 1]
```diff
@@ -64,6 +64,9 @@ export const useMutateAccount = () => {
                 variant: 'info',
               }
             );
+            queryClient.invalidateQueries({
+              queryKey: accountQueries.notifications.queryKey,
+            });
```
Comment (bnussman-akamai) [inline packages/manager/src/queries/account/account.ts:69]: Shouldn't be needed, right?

Comment (jaalah-akamai) [reply packages/manager/src/queries/account/account.ts:69]: Initially we were able to rely on events here https://github.com/linode/manager/blob/develop/packages/manager/src/queries/account/billing.ts#L36-L40 but those events are sent until after verification

**Pull request merged**